src/main/fc/rc.c

```diff
@@ -82,14 +83,15 @@ enum {
 };
 
 #ifdef USE_RC_SMOOTHING_FILTER
+#define RC_SMOOTHING_CUTOFF_MIN_HZ              15    // Minimum rc smoothing cutoff frequency
 #define RC_SMOOTHING_FILTER_STARTUP_DELAY_MS    5000  // Time to wait after power to let the PID loop stabilize before starting average frame rate calculation
 #define RC_SMOOTHING_FILTER_TRAINING_SAMPLES    50    // Number of rx frame rate samples to average during initial training
 #define RC_SMOOTHING_FILTER_RETRAINING_SAMPLES  20    // Number of rx frame rate samples to average during frame rate changes
 #define RC_SMOOTHING_FILTER_TRAINING_DELAY_MS   1000  // Additional time to wait after receiving first valid rx frame before initial training starts
 #define RC_SMOOTHING_FILTER_RETRAINING_DELAY_MS 2000  // Guard time to wait after retraining to prevent retraining again too quickly
 #define RC_SMOOTHING_RX_RATE_CHANGE_PERCENT     20    // Look for samples varying this much from the current detected frame rate to initiate retraining
-#define RC_SMOOTHING_RX_RATE_MIN_US             1000  // 1ms
-#define RC_SMOOTHING_RX_RATE_MAX_US             50000 // 50ms or 20hz
+#define RC_SMOOTHING_RX_RATE_MIN_US             950   // 0.950ms to fit 1kHz without an issue
```

Comment:
This is great - thanks! While we're at it, the calculation of currentRxRefreshRate at line 296 constrains the minimum to a hard coded 1000us. I've had to reduce this in the past to be able to use the debug to check the jitter on elrs. Could we fix it at the same time? https://github.com/betaflight/betaflight/blob/6d286e25f14f1b1bb977c2f17ffa42c3ffe985f3/src/main/fc/rc.c#L296

Reply:
It is already fixed, here:
https://github.com/betaflight/betaflight/pull/10832/files#diff-72d835b20e40a1a7a44822a8142ee40c31c87a7628d5d544aadd1ab6be4ee258R304

Reply:
Already fixed. Marking as resolved.

Reply:
ah, sorry, I should have looked harder. many thanks!